altair/utils/_show.py

```diff
@@ -0,0 +1,75 @@
+import webbrowser
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from typing import Union, Iterable, Optional
+
+
+def open_html_in_browser(
+    html: Union[str, bytes],
+    using: Union[str, Iterable[str], None] = None,
+    port: Optional[int] = None,
+):
+    """
+    Display an html document in a web browser without creating a temp file.
+
+    Instantiates a simple http server and uses the webbrowser module to
+    open the server's URL
+
+    Based on
```

Comment:
Incomplete sentence

Reply:
Done in 478944342eff67dedc49fe699252d97d680e07bc